### PR TITLE
Revert "Bump github.com/onsi/gomega from 1.14.0 to 1.18.0"

### PR DESCRIPTION
**Pull request by dlresende**
Reverts cloudfoundry/smbbroker#122

### CODE REVIEW

**Comment by dlresende** [comment]
Sorry, I've clicked some buttons on the Github UI and our automation merged #122 (not what I intended).

**Comment by dlresende** [comment]
tl;dr With the context that I've gathered, I'm of the opinion that we should not add this commit.

I believe the commit aims to make some connections this component establishes more secure. I see this as a new feature and I believe it goes beyond the scope of [Cryogenics](https://github.com/orgs/cloudfoundry/teams/cryogenics).

To be fair I don't think we need to be that strict. But in this case, given the various existing and supported versions of platforms (like TAS) and components (like Credhub, etc), I'm unable to call out whether this would break connections without testing some combinations of versions.

But if we were to merge, I'd rather have it on its own PR.

**Comment by fejnartal** [comment]
@dlresende we will have to manually merge this PR. Our CI will refuse to merge it automatically because gosec detects this issue:
https://deepsource.io/gh/vagababov/networking/issue/GSC-G402/description/

An alternative would be to disable this specific check for the repository as implemented here:
https://github.com/RedHatInsights/insights-operator-controller/pull/381/files

The advantage of disabling it is that any future dependency bumps would be able to be tested and merged automatically.

**Comment by fejnartal** [comment]
I have created a new PR which should replace this one:
- Disables gosec rule G402 TLS MinVersion too low
- Removes parameter `MinVersion: tls.VersionTLS12,` that was merged recently
- Keeps the updated version of gomega (instead of reverting everything)

https://github.com/cloudfoundry/smbbroker/pull/124

**Comment by dlresende** [comment]
LGTM.
